### DISCUSSION TITLE
I literally hate "The Feed"

### DISCUSSION THREAD

**Discussion by mcclure** [community/community#54017] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why does this exist. This is a page of things I don't want to look at. There was exactly one thing in "the Feed" that was a project I would ever consider using, and it was the opposite of helpful to know someone had pushed a random commit to it. The others are all chatgpt garbage. I have never interacted with an "AI" project and I have no idea why your "Feed" thinks I want to see an entire page of them. All you're doing is reminding me how angry I am that Microsoft copilot stole all my source code and is making derivative works out of them in contravention of the licenses. Like literally every project in "The Feed" was something I would block if you have a block button. Is there a block button? Can I block "The Feed"?

There was an offer at the top of "the Feed' that I could provide feedback. My feedback is not only should you not show this to me anymore you should not show it to anyone. I got on the site to do some work but now instead of getting things done 'm going to be wasting time in the preferences trying to figure out if I can turn off "The Feed". I am struggling to not use cuss words here.

**Comment by mcclure** [comment]
After failing to find an off switch for "the Feed" in Settings, I searched Google and discovered there is a tab at the top of the feed to switch from the "here are a bunch of AI projects" tab to the "here are some things that you have actually participated in" tab. I did not notice this at first because I was on my phone so the tab switcher was small and also the moment I saw the contents of the "For you" tab I immediately wanted nothing more on earth than to leave the page and never look at it again.

I am not marking this as an answer because I now 1000%.expect you're going to do that Twitter/Facebook thing of switching me back to the "For You" tab monthly and making me manually switch back each time. You should offer the ability to fully disable the "for you" tab or, even better, remove the "for you" feature from the site completely

**Comment by gostega** [comment]
I feel the same way. I want to log into github and see my repositories like normal. I get a 'Feed' of things happening on github from the other places I visit, like youtube, twitter, my email, and blogs I follow etc. I don't want this at all or anything like it.

**Comment by zautumnz** [comment]
The Twitter-ification of every site and its consequences have been a disaster for internet users. Turns out that building a recommendation system is not easy and most users, especially those technically inclined, prefer a plain reverse-chronological feed. Instagram's post interactions dropped 40% when they removed this in 2015 (they ended up bringing it back, eventually). A recommendations-based feed makes perfect sense for ecommerce and some social media systems, but on a site like GitHub it honestly feels like a waste of developer time.

One other problem: this setting seems to either be reset once in a while, or persist in a cookie or local storage, because every so often mine switches back to the Feed.

**Reply by mcclure** [reply]
Yeah, yesterday (just a few days after posting this!) I opened up a browser to look at github and I was looking at the loathsome "for you" feed again despite specifically having turned it off. Just as I feared, Github will not actually let you switch away. We really need, at minimum, an account-level setting to disable "for you" entirely.

**Reply by csilverman** [reply]
If you have a browser plugin that lets you apply custom CSS to websites (like Chrome's [Stylebot](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha)), the following should do the trick:

```
#dashboard-feed-frame div#panel-2 {
    display: none;
}

#dashboard-feed-frame div#panel-1[hidden][hidden] {
    display: block !important;
}

#dashboard-feed-frame .js-feeds-tabs li.d-inline-flex:first-child {
    display: none !important;
}
```

I hope GitHub comes to its senses and scraps this stupidity entirely, but in the event that they don't, it seems to be easy to remove at this point.

**Reply by zautumnz** [reply]
@Gitvinls do you really think GitHub issue comments are vulnerable to xss?

**Comment by Foggzie** [comment]
Just wanted to +1 this with a comment.  I absolutely **_hate_** any idea of a "feed"; I'm sick of being fed content. Github is a **_TOOL_** for me, not another social network, not another place to consume nor share news. Let me disable this entirely. If I clear out my filter list, then 80% of the homepage turns into blank space dedicated to a feature I want nothing to do with. Why is this absolutely **_wretched_** feature is given so much real estate!?

![image](https://user-images.githubusercontent.com/1359423/236510994-4ee5db1a-50f6-4c65-a7ed-f95052d15c1a.png)

**Reply by csilverman** [reply]
Agreed. I'm guessing this is the Microsoft side of things creeping into GitHub, because anybody who uses and understands GitHub would have known that trying to shoehorn one of Twitter's most reviled features into GitHub was going to result in a shitshow.

Really hope GH gives a damn about their users, because this isn't a mere "eh, not really into it" thing; as far as I can tell, people hate this crap on a visceral level.

**Comment by mgraybosch-deloitte** [comment]
The "Feed" is absolutely worthless to me, and should be **opt-in** only. As it currently stand I don't even see an **opt-out** switch. This is unacceptable.

I only use GitHub at work. I only want to see repositories that I've flagged as relevant to my work. I do *not* want a feed full of irrelevant garbage, and *nobody* is paying me enough to deal with bullshit (as defined by Harry Frankfurt) generators like ChatGPT.

This sort of "Let's make GitHub yet another social media platform," foolishness will not persuade me to use GitHub when I'm not getting paid. I will instead stick to SourceHut for personal projects. And if I *really* wanted a social platform for developers, I know where to find https://dev.to; I don't frequent that site either because I don't get paid enough to put up with social media.

**Comment by nyghtly-derek** [comment]
They should absolutely remove this feature! In the short term, you can block the feed yourself by using an adblocker. Most adblocker extensions have a feature that let you to point-and-click on page elements that you want to hide. 

For example, using uBlock Origin:
![image](https://user-images.githubusercontent.com/7283262/236552978-d2c7ac8a-ed2b-4ff6-82e4-ee913ff3405c.png)

And here is the config I landed on:
```
! 2023-05-06 https://github.com
github.com##.js-for-you-feed-items
github.com##.d-flex.feed-content.border-bottom.mt-3
```

**Comment by StephanieSunshine** [comment]
This feed is killing me. I'm just trying to find my repositories. I don't need a news feed. if I wanted a news feed I would goto Reddit. This is distracting. How do I opt out of this crap?

**Comment by JasonSafro** [comment]
Yes. Duck your ducking feed. Please:

1. find the manager who thought github needed to be social media
2. kick him in the testicles
3. post that on the ducking news feed
